### PR TITLE
Lisna Anya

### PR DESCRIPTION
**Pull request by AnyaLisna**

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+class Numberdrome {
+
```
Comment (gestap) [inline src/script.js]: remove empty line

File: src/script.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    constructor() {
```
Comment (gestap) [inline src/script.js]: use 2 whitespaces instead of 4

File: src/script.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    sum() {
+       return this.numbers.reduce((sum, number) => sum + number, 0);
+    }
+
+    product() {
+        return this.numbers.reduce((sum, number) => sum * number, 1);
+    }
+
+    min() {
+        return Math.min(...this.numbers);
```
Comment (gestap) [inline src/script.js]: `... or throws an error if there are no numbers;`

File: src/script.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    max() {
+        return Math.max(...this.numbers);
```
Comment (gestap) [inline src/script.js]: `... or throws an error if there are no numbers`

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+
+  <div id='container'></div>
+
+  <script src="script.js"></script>
+</body>
+</html>
```
Comment (gestap) [inline src/index.html]: the document must end with one empty line

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**